opencompass/datasets/notdiamond/gpqa.py

```diff
@@ -0,0 +1,81 @@
+import copy
+import csv
+import os
+
+from datasets import Dataset
+
+from opencompass.openicl import BaseEvaluator
+from opencompass.registry import LOAD_DATASET
+
+from .base import BaseDataset
+
+
+@LOAD_DATASET.register_module()
+class GPQADataset(BaseDataset):
```

Comment:
This needs to be renamed to `NDGPQADataset` because the module register can only accept unique class names